src/views/common/TimeScrollView/TimeScrollViewInteractions/TimeScrollViewEventHandlers.ts

```diff
@@ -0,0 +1,113 @@
+import { useTimeFocus, useTimeRange } from 'contexts/RecordingSelectionContext';
+import React, { useCallback } from 'react';
+import { clearDivFocus, divExists, setDivFocus } from './divRefHandling';
+import useTimeScrollPan, { PanUpdateProperties } from './useTimeScrollPan';
+
+
+export const suppressWheelScroll = (divRef: React.MutableRefObject<HTMLDivElement | null>) => {
+    if (!divRef || !divRef.current) return
+    const canvases = Array.from(divRef.current.children).filter(e => e.nodeName === 'CANVAS')
+    canvases.forEach(c => {
+        c.addEventListener('wheel', (e: Event) => {
+            if ((divRef?.current as any)['_hasFocus']) {
+                e.preventDefault()
+            }
+        })
+    })
+}
```

Comment:
This guy really could go somewhere else as we might want to reuse it, but I'm not fussed for now. (For instance it should probably be run on the animation component, since that also uses mouse wheel handlers...)